src/schneider_model.cpp

```diff
@@ -60,6 +62,12 @@ void Schneider::init() {
 
 void Schneider::one_step() {
     using std::abs;
+    // ジョイスティックの値の実効下限値
+    constexpr auto joy_min = 0.4F;
+    // つまみの値の実効範囲 (x < volume_under || volume_over < x)

```

Comment:
```suggestion
    // つまみの値の実効範囲 (x < volume_under || volume_over < x) で有効
```

条件式書いてあるだけだとどっちがどっちがわからないので